### PR TITLE
add pypy3

### PR DESCRIPTION
**Pull request by noviluni**
Draft PR to see if the pypy3 is passing the pipelines

### CODE REVIEW

**Comment by noviluni** [comment]
* Results when adding Pypy3 and pinning the regex package version: https://travis-ci.org/github/scrapinghub/dateparser/jobs/709075900
* Results when adding Pypy3 without pinning the regex package version: https://travis-ci.org/github/scrapinghub/dateparser/jobs/709080421

**Comment by Gallaecio** [comment]
It looks like the non-pinned run fails due to https://bitbucket.org/pypy/pypy/issues/2723/undefined-symbol-pyunicode_new-in-pypy3

I think the issue is that we are running an old version of PyPy. If you have a look at Scrapy’s Travis and Tox configuration, you’ll see how to run the latest version (we’ve merged code for that this very week).

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ matrix:
       env: TOXENV=py38
     - python: 3.9-dev
       env: TOXENV=py39
+    - python: pypy3
+      env: TOXENV=pypy3 PYPY_VERSION=3.5-v7.0.0
```
Comment (Gallaecio) [inline .travis.yml:20]: `PYPY_VERSION` is not something Travis understands. [We use it in Scrapy to manually install that version before the tests start](https://github.com/scrapy/scrapy/blob/master/.travis.yml#L49-L57).

**Comment by noviluni** [comment]
Hi @Gallaecio, I was checking the Scrapy pipelines but I didn't get the point. Could you help me to configure the pipelines?

File: .travis.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ matrix:
...
 
 install: pip install -U wheel tox codecov
```
Comment (Gallaecio) [inline .travis.yml:22]: This should be enough to make `PYPY_VERSION` work:

```suggestion
install:
  - |
      if [[ ! -z "$PYPY_VERSION" ]]; then
        export PYPY_VERSION="pypy$PYPY_VERSION-linux64"
        wget "https://bitbucket.org/pypy/pypy/downloads/${PYPY_VERSION}.tar.bz2"
        tar -jxf ${PYPY_VERSION}.tar.bz2
        virtualenv --python="$PYPY_VERSION/bin/pypy3" "$HOME/virtualenvs/$PYPY_VERSION"
        source "$HOME/virtualenvs/$PYPY_VERSION/bin/activate"
      fi
  - pip install -U wheel tox codecov
```

**Comment by Gallaecio** [comment]
It did not help.

Given tests do not pass, I’m +1 to dropping support in the next version, until someone comes with a way to restore support.

**Comment by noviluni** [comment]
thanks for your feedback @Gallaecio! 

I added a note in the issue regarding this: https://github.com/scrapinghub/dateparser/issues/745#issuecomment-680075168

Thank you again! :smile:

**Pull request closed without merging**